docs/OV_Runtime_UG/deployment/conditional_compilation_deployment.md

```diff
@@ -0,0 +1,289 @@
+
+            #pragma once
+
+            #define tbb_bind_TBBbindSystemTopology 1
+            #define tbb_bind_task_arena_initialize 1
+
+            #define ov_opset_opset1_Parameter 1
+            #define ov_opset_opset1_Constant 1
+            #define ov_opset_opset1_Convolution 1
+            #define ov_opset_opset1_Add 1
+            #define ov_opset_opset1_Relu 1
+            #define ov_opset_opset1_GroupConvolution 1
+            #define ov_opset_opset3_ShapeOf 1
+            #define ov_opset_opset1_Squeeze 1
+            #define ov_opset_opset4_Range 1
+            #define ov_opset_opset1_ReduceMean 1
+            #define ov_opset_opset1_Softmax 1
+            #define ov_opset_opset1_Result 1
+
+            #define ov_op_v0_Parameter_visit_attributes 1
+            #define ov_op_v0_Parameter_validate_and_infer_types 1
+            #define ov_op_v0_Parameter_clone_with_new_inputs 1
+            #define ov_op_v0_Constant_visit_attributes 1
+            #define ov_op_v0_Constant_clone_with_new_inputs 1
+            #define ov_op_v1_Convolution_visit_attributes 1
+            #define ov_op_v1_Convolution_validate_and_infer_types 1
+            #define ov_op_v1_Convolution_clone_with_new_inputs 1
+            #define ov_op_v0_util_BinaryElementwiseArithmetic_visit_attributes 1
+            #define ov_op_v1_Add_visit_attributes 1
+            #define ov_op_v0_util_BinaryElementwiseArithmetic_validate_and_infer_types 1
+            #define ov_op_v1_Add_clone_with_new_inputs 1
+            #define ov_op_v0_Relu_visit_attributes 1
+            #define ov_op_util_UnaryElementwiseArithmetic_validate_and_infer_types 1
+            #define ov_op_v0_Relu_clone_with_new_inputs 1
+            #define ov_op_v1_GroupConvolution_visit_attributes 1
+            #define ov_op_v1_GroupConvolution_validate_and_infer_types 1
+            #define ov_op_v1_GroupConvolution_clone_with_new_inputs 1
+            #define ov_op_v3_ShapeOf_visit_attributes 1
+            #define ov_op_v3_ShapeOf_validate_and_infer_types 1
+            #define ov_op_v3_ShapeOf_clone_with_new_inputs 1
+            #define ov_op_v0_Squeeze_visit_attributes 1
+            ...
+
+
+Stage 2: build resulting OpenVINO package
+++++++++++++++++++++++++++++++++++++++++++
+
+Based on the statistics information, re-build OpenVINO to generate the optimal binary size of OpenVINO binaries
+
+.. code-block:: sh
+
+    cmake -DSELECTIVE_BUILD=ON -DSELECTIVE_BUILD_STAT=${ABSOLUTE_PATH_TO_STATISTICS_FILES}/*.csv -DENABLE_PROFILING_ITT=OFF ..
+    cmake --build .
+
+.. tip::
+
+    The recommended scenario for conditional complication is static build of OpenVINO, in this case you can add ``-DBUILD_SHARED_LIBS=OFF`` to enable static build to get optimal binary size benefit.
+
+
+Conditional compilation for different ISAs
+##########################################
+
+It is almost the same steps with the build for different models except for collecting different statistics data on different ``ISAs``.
```

Comment:
please provide full abbreviations when they are first mentioned like this - Instruction Set Architectures (ISAs), same for ITT, SCU, SDE, etc.